sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryListenerSuite.scala

```diff
@@ -48,8 +48,7 @@ class StreamingQueryListenerSuite extends StreamTest with BeforeAndAfter {
     spark.streams.active.foreach(_.stop())
     assert(spark.streams.active.isEmpty)
     // Skip check default `StreamingQueryStatusListener` which is for streaming UI.
-    assert(spark.streams.listListeners()
-      .filterNot(_.isInstanceOf[StreamingQueryStatusListener]).isEmpty)
+    assert(spark.streams.listListeners().forall(_.isInstanceOf[StreamingQueryStatusListener]))
```

Comment:
Can we have a newline to match with https://github.com/apache/spark/pull/31188/files#diff-ea70ab4d37b86e958c50f0d0eb0f243c2eb3312c3c0afc78576fa981e5a5a7bbR52?

Reply:
ok

Reply:
Address 4ac192b add a newline